src/reference.yaml

```diff
@@ -48,6 +48,8 @@ tags:
       and collect funds in each available currency
   - name: IBANs
     description: >-
+      **This endpoint is deprecated and will be removed at the end of February. Please use the Funding Account API instead.**
```

Comment:
@maria-thom I you think you should add the year as well (February 2021)

Reply:
@jonathancouchman Agreed - added the year.